### PR TITLE
feat(#1820): remove tuple.empty object

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Remove `tuple.empty` object.

Closes: #1820

---

## PR-Codex overview
This PR removes the `empty` method from the `tuple` object and introduces fluent syntax to create tuples. It also adds new tests for the updated functionality.

### Detailed summary
- Removes the `empty` method from the `tuple` object
- Introduces fluent syntax to create tuples
- Adds new tests for the updated functionality.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@Graur could you have a look, please?

**Comment by Graur** [comment]
@volodya-lombrozo  As far as I know, you may be able to create a `tuple` object by simply calling its name. Like:

```
[] > test
  tuple > a
  a.with 3 > @
```

Looks like this pull request can't solve this issue

**Comment by volodya-lombrozo** [comment]
@Graur The PR is ok now, could you take a look, again, please?

**Comment by Graur** [comment]
@volodya-lombrozo One possible way is to create `@` atom inside `tuple` object. Like the following:

```
[] > tuple
  [] > @ /?
...
```
and implement empty array initialisation logic in `EOtuple$EO@.java` class

Or maybe it will be possible to create an empty array in runtime. E.g. in `to-java.xsl`.

**Comment by volodya-lombrozo** [comment]
Non of the next improvements don't help:
```
[] > tuple
  [] > @ /?
...
```
or
```
[] > tuple
  * > @
...
```
or
```
[] > tuple
  [] > @
    * > @
```

File: eo-runtime/src/test/eo/org/eolang/tuple-tests.eo [hunk 1]
```diff
@@ -183,13 +183,30 @@
...
+
+[] > creates-empty-tuple-with-number
+  tuple.with 3 > a
+  assert-that > @
+    a.at 0
+    $.equal-to 3
+
 [] > tuple-fluent-with-indented-keyword
-  tuple.empty
+  *
```
Comment (Graur) [inline eo-runtime/src/test/eo/org/eolang/tuple-tests.eo:209]: @volodya-lombrozo Should `tuple` be here?

**Review by Graur — CHANGES_REQUESTED** [review]
@volodya-lombrozo Indeed, triggering the creation of a `tuple` object at runtime was a bad idea on my part.

**Comment by volodya-lombrozo** [comment]
@Graur Could you take a look again, please?

**Review by Graur — APPROVED** [review]
@volodya-lombrozo I like this solution!
@yegor256 Please, have a look

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you merge that changes, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**